packages/api/src/APIClient.ts

```diff
@@ -284,6 +301,76 @@ export default class APIClient {
 			),
 	}
 
+	public readonly otherIncomes = {
+		/**
+		 * Get list of other incomes
+		 */
+		list: (
+			accountId: string,
+			queryBuilders?: QueryBuilderType[]
+		): Promise<Result<{ otherIncomes: OtherIncome[]; pages: Pagination }>> =>
+			this.call(
+				'GET',
+				`/accounting/account/${accountId}/other_incomes/other_incomes${joinQueries(queryBuilders)}`,
+				{
+					transformResponse: transformListOtherIncomesResponse,
+				},
+				null,
+				'List OtherIncomes'
+			),
+		/**
+		 * Get single other income
+		 */
+		single: (
+			accountId: string,
+			otherIncomeId: string,
+			queryBuilders?: QueryBuilderType[]
+		): Promise<Result<OtherIncome>> =>
+			this.call(
+				'GET',
+				`/accounting/account/${accountId}/other_incomes/other_incomes/${otherIncomeId}${joinQueries(queryBuilders)}`,
+				{
+					transformResponse: transformOtherIncomeResponse,
+				},
+				null,
+				'Get OtherIncome'
+			),
+		/**
+		 * Post other income
+		 */
+		create: (otherIncome: OtherIncome, accountId: string): Promise<Result<OtherIncome>> => {
```

Comment:
Maybe `accountId` comes before `otherIncome` parameter like we discussed.

Reply:
@seanmoon80 this is based on other `create` endpoints and all of them have it another way around so I think for the user, for now, is better if we keep it in that way